src/components/chip-set/chip-set.tsx

```diff
@@ -347,6 +353,16 @@ export class ChipSet {
         }, 0);
     }
 
+    private syncEmptyInput() {
+        this.textValue = '';
+
+        if (this.value && this.value.length) {
+            // If there are chips in the picker, hide the input to avoid the input
+            // being placed on a new line and adding ugly space beneath the chips.
+            this.inputHidden = true;
```

Comment:
It seems like the input is sometimes hidden when it shouldn't be. I'll need look further into this.

Reply:
⚡️